src/entity.ts

```diff
@@ -1249,34 +1238,12 @@ export namespace entity {
       queryProto.startCursor = query.startVal;
     }
 
-    if (query.filters.length > 0) {
-      const filters = query.filters.map(filter => {
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        let value: any = {};
-
-        if (filter.name === '__key__') {
```

Comment:
why are we removing this `__key__` logic?

Reply:
This OR query implementation created duplicate code so this code has not been removed, it has only been refactored into the `PropertyFilter` object.

Take a look at https://github.com/googleapis/nodejs-datastore/pull/1061/files#diff-87404c91a22dbabcc334f9a5288341281cb2b6625533e1207efcac16dc1e2ffdR77.